docs/changelog.md

```diff
@@ -178,10 +209,10 @@ compile "org.janusgraph:janusgraph-core:1.0.1"
 * Apache HBase 2.5.8
```

Comment:
**❓ question:** ‏I think that should be 2.6.0 based on [this line](https://github.com/porunov/janusgraph/blob/release/v1.1.0/pom.xml#L71)

Reply:
Same, that is for Version 1.0.1 and not 1.1.0. Several lines above description for 1.1.0 contains `2.6.0` as you specified

Reply:
Sorry about that!

Reply:
No worries. Thank you!
I will be merging this PR only after 1.0.1 PR #4722 
So that dates are relevant